### ISSUE TITLE
Helpers are inlined when using 'externalHelpers'

### ISSUE THREAD

**Issue by aparajita** [rollup/rollup-plugin-babel#295]
The bug reported in #245 remains. Using a global `babelHelpers` (as documented in this plugin's README) is currently not possible, and of course should be.

Here's the problem:

https://github.com/rollup/rollup-plugin-babel/blob/e74683232e2b9fc15199e890074a6151cfa5c55c/src/index.js#L108-L110

Line 108 seems wrong.

- If I run the code through `babel`, it's doing the right thing: generating `babelHelpers.classCallCheck`, etc.
- I've got `externalHelpers` set to `true` in the rollup config.
- I've checked the result of `preflightCheck` and it's correctly returning `EXTERNAL`.

But because line 108 does not check if `helpers` is `EXTERNAL`, an import is being added to the generated code that results in the helpers being inserted at the top of the generated code. If I change line 108 to:

```js
if (helpers === INLINE) {
```

then the code is correct, the calls to the external `babelHelpers` remain.

**Comment by Andarist** [comment]
Could u prepare a sample repo showcasing the problem? It would help tremendously as I could get into fixing this quicker.

**Comment by aparajita** [comment]
Sure, I'll put something together today.

**Comment by bdwain** [comment]
I noticed this too, and it seems to have started when I upgraded to babel7 and rollup-plugin-babel 4.0.3 from v3.0.7.

**Comment by aparajita** [comment]
Yes, it's a regression that appeared somewhere during that upgrade.

**Comment by aparajita** [comment]
Here's a reduction.

```bash
npm i
gulp compile
gulp compile-fixed
```

Notice that `gulp compile` generates the inlined helpers, whereas `gulp compile-fixed` uses the global `babelHelpers`. The only difference between them is I'm using a fixed `rollup-plugin-babel` that changes the line I mentioned.
[issue-295.zip](https://github.com/rollup/rollup-plugin-babel/files/2775888/issue-295.zip)

**Comment by Andarist** [comment]
Fixed in 4.3.2, could you confirm it?

**Comment by aparajita** [comment]
Fixed, thank you!
